README.md

```diff
@@ -13,13 +13,20 @@
 
 # Enceladus
 
+### <a name="latest_release"/>Latest Release
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa.enceladus/parent/badge.png)](https://maven-badges.herokuapp.com/maven-central/za.co.absa.enceladus/parent/)
+
 ### <a name="build_status"/>Build Status
 | master | develop |
 | ------------- | ------------- |
 | [![Build Status](https://opensource.bigusdatus.com/jenkins/buildStatus/icon?job=Absa-OSS-Projects%2Fenceladus%2Fmaster)](https://opensource.bigusdatus.com/jenkins/job/Absa-OSS-Projects/job/enceladus/job/master/) | [![Build Status](https://opensource.bigusdatus.com/jenkins/buildStatus/icon?job=Absa-OSS-Projects%2Fenceladus%2Fdevelop)](https://opensource.bigusdatus.com/jenkins/job/Absa-OSS-Projects/job/enceladus/job/develop/) |
 ### <a name="code_quality_status"/>Code Quality Status
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=AbsaOSS_enceladus&metric=alert_status)](https://sonarcloud.io/dashboard?id=AbsaOSS_enceladus)
 
+### <a name="documentation"/>Documentation
+[![Read the Docs](https://img.shields.io/badge/docs-latest-brightgreen.svg)](https://absaoss.github.io/enceladus/)
+[![Read the Docs](https://img.shields.io/badge/docs-release%20notes-yellow.svg)](https://absaoss.github.io/enceladus/blog/)
```

Comment:
Not sure about the colors here:
Green for current docs, yellow for release notes, red for obsolete docs
Wouldn't green-green-yellow be better/nicer/make more sense?

Reply:
See at https://github.com/AbsaOSS/enceladus/tree/project/project-config-and-management-updates